.gitignore

```diff
@@ -22,6 +22,7 @@ robot/UST-EASY/results/
 
 # Misc
 .DS_Store
+James Scratch Org Notes
```

Comment:
Personalization in an enterprise solution? I'll let it pass.

Reply:
I'll hide it in a professional-sounding folder next time.